### PR TITLE
Write lab results to contacts

### PR DESCRIPTION
**Pull request by ljwoodley**
I emailed you a fake dataset to test the transformations . Also, should this script be renamed to write_lab_results as it's writing to cases, contacts and results now.

 Addresses issue #151

### CODE REVIEW

File: R/data_creating.R [hunk 1]
```diff
@@ -316,3 +318,76 @@ get_rejected_lab_data <- function(raw_lab_result_data,
...
+        redcap_event_name,
+        case_id,
+        contact_ll_ufid,
+        cleared_for_campus,
+        contains("_v"),
+        contact_lab_collect_date = collection_date,
+        contact_lab_report_date = result_date,
+        contact_lab_result = results
+      )
+
```
Comment (ljwoodley) [inline R/data_creating.R:350]: Do `new_contact_labs <- fake_contact_labs` here for testing

**Comment by pbchase** [comment]
> ... Also, should this script be renamed to write_lab_results as it's writing to cases, contacts and results now.

Yes, that's a better name. please proceed with the rename.

**Comment by pbchase** [comment]
You'll need to revise the name in the cron job as well.

**Comment by ljwoodley** [comment]
As I push the case_id change I realized `record_id` is already there. That is unique even when contact_ll_ufid is duplicated so the case_id is not needed. I'll leave it as it's already committed.

**Comment by pbchase** [comment]
Beautiful work. :-)  Merging.

**Pull request merged**